src/components/divider/divider.styles.ts

```diff
@@ -0,0 +1,5 @@
+import { SxStyleProp } from 'rebass';
+
+export default {
+  borderBottom: '2px solid #F5F5F5',
```

Comment:
Use the variable for color not the absolute value

Reply:
added label!